bin/v-change-sys-ip-status

```diff
@@ -35,6 +35,9 @@ ip_owner=$(get_ip_value '$OWNER')
 if [ "$web_domains" -ne '0' ] && [ "$sys_user" != "$ip_owner" ]; then
     check_result "$E_INUSE" "ip $ip is used"
 fi
+if [ "$ip_owner" != "admin" ]; then
```

Comment:
`if [ "$ip_owner" != "admin" ] && [ "$ip_status"="shared" ]; then`
So it won't abort when switching ipstatus from shared to dedicated on ips incorrectly configured as shared previously

Reply:
should be good now.